microsoft-365/security/mtp/top-scoring-industry-tests.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Top scoring in industry tests (AV-TEST, AV Comparatives, SE Labs, MITRE ATT&CK)
+title: Microsoft Threat Protection - Top scoring in industry tests
 ms.reviewer: 
-description: Microsoft Defender ATP consistently achieves high scores in independent tests. View the latest scores and analysis.
+description: Learn why Microsoft Defender ATP consistently achieves high scores in independent av tests (AV-TEST, AV Comparatives, SE Labs, MITRE ATT&CK). View the latest scores and analysis.
```

Comment:
Microsoft Defender ATP should actually be updated to Microsoft Threat Protection. And it's not just AV tests anymore.

"Learn why Microsoft Threat Protection consistently achieves high scores in independent tests (AV-TEST, AV Comparatives, SE Labs, MITRE ATT&CK). View the latest scores and analysis.